### DISCUSSION TITLE
Connection issue Win 11 w/ 2000mW mini laser engraver

### DISCUSSION THREAD

**Discussion by Banach95** [arkypita/LaserGRBL#1906] [Q&A]
Ok complete newbie here. I bought a "LEEPRA Purple 2000mW Mini Laser Engraver Printer DIY Logo Marking USB Engraving Machine 8x8cm" (that was a mouthful... holy keyword spamming) off of Amazon back in 2018. The seller doesn't exist anymore and I've lost the thumb drive with the program/drivers on it. 

I've done some research and it looks like it was a mass produced model that still exists today under different company names. Which brought me to LaserGRBL to try. I have surmised that there is a driver issue and it's probably because I'm running Win 11 and not Win 10. I read through the help section and tried installing the ComPort drivers listed to no avail.

There is electric power via the power cord (pushing the buttons on the machine manually moves the laser) but when inserting the USB cord to the laptop nothing happens and there's no connection. Now prior to upgrading to Win 11; this setup worked on this laptop.

Please be gentle in bashing me LOL. I learn from every interaction with forums like this; so aside from downgrading back to Win 10 or Win 7 (which is what I ran when I first got this) what could I do to try to get a connection to the engraver?

**Comment by Tribo-2dot0** [comment]
O Driver CH340 aparece no Gerenciador de Dispositivos quando a máquina está plugada?

**Comment by Banach95** [comment]
No

These is what I see with the USB from the machine direct into my laptop
![20220824_152028](https://user-images.githubusercontent.com/111715830/186505267-d8da4de1-74c3-4261-8172-36ea27de6abe.jpg)
![20220824_152043](https://user-images.githubusercontent.com/111715830/186505276-0aecb8b1-9444-46ea-aea1-a72ba5c6ab37.jpg)

**Comment by Tribo-2dot0** [comment]
Tens que instalar o driver CH340... mas não sei se esse tipo de máquina suporta LaserGrbl ou qualquer outro software do tipo, tenho um Neje DK-8-KZ e só aceita o software dele (e um outro semelhante)... é uma maquina que nao faz vetor.
